stackslib/src/chainstate/stacks/auth.rs

```diff
@@ -314,6 +315,204 @@ impl MultisigSpendingCondition {
     }
 }
 
+impl StacksMessageCodec for OrderIndependentMultisigSpendingCondition {
+    fn consensus_serialize<W: Write>(&self, fd: &mut W) -> Result<(), codec_error> {
+        write_next(fd, &(self.hash_mode.clone() as u8))?;
+        write_next(fd, &self.signer)?;
+        write_next(fd, &self.nonce)?;
+        write_next(fd, &self.tx_fee)?;
+        write_next(fd, &self.fields)?;
+        write_next(fd, &self.signatures_required)?;
+        Ok(())
+    }
+
+    fn consensus_deserialize<R: Read>(
+        fd: &mut R,
+    ) -> Result<OrderIndependentMultisigSpendingCondition, codec_error> {
```

Comment:
See this discussion from September ([link](https://github.com/stacks-network/stacks-core/pull/3710#discussion_r1330289607)). I made the same suggestion and @fess-v implemented it, but we later reverted it because you didn't want to risk making a change to consensus-critical code

Reply:
Ah, you're right.  My past self once again saves my current self from being reckless